### ISSUE TITLE
T-SNE Embedding 

### ISSUE THREAD

**Issue by redhat12345** [HiGal/Domain-Adaptation#1]
I run your code but do not get any t-SNE visualization. Can you please upload the code for t-SNE visualization?

**Comment by HiGal** [comment]
Yes, of course as soon as possible I will upload it) It just a few lines of code)

**Comment by bohemian98** [comment]
Just waiting for the t-SNE visualization code.
